examples/with-okta-multi-tenant-pkce-flow/api-server/config.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func getRequestFromUserContext(userContext supertokens.UserContext) *http.Request {
+	if _default, ok := (*userContext)["_default"].(map[string]interface{}); ok {
+		if request, ok := _default["request"].(*http.Request); ok {
+			return request
+		}
+	}
+	return nil
+}
+
+func getTenantIdFromUserContext(userContext supertokens.UserContext) string {
```

Comment:
change this function to be getTenantIdFromRequest

Reply:
done